### PR TITLE
Knowledge update:  Finishing domain 1

### PR DESCRIPTION
**Pull request by coppolaop**

### CODE REVIEW

**Review by Flaigor — APPROVED** [review]
it's ok!

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 We are [Marcos](https://github.com/coppolaop) and [Igor](https://github.com/flaigor)
 
 ## Our certification topics
-- [ ] Domain 1: Introduction to Git and GitHub (22% of the test)
```
Comment (Flaigor) [inline README.md:5]: this part is ok!

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
-- - [ ] Git and GitHub Basics
+- [x] Domain 1: Introduction to Git and GitHub (22% of the test)
```
Comment (Flaigor) [inline README.md:5]: ok

Comment (Flaigor) [reply README.md:5]: test

File: README.md (hunk 1 above)
Comment (Flaigor) [reply README.md:5]: test 2

Comment (Flaigor) [reply README.md:5]: @coppolaop teste

**Review by Flaigor — APPROVED** [review]
Approve again!

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ We are [Marcos](https://github.com/coppolaop) and [Igor](https://github.com/flai
 - - - [x] Describe a commit
 - - - [x] Describe branching
 - - - [x] Define a remote in Git terminology
-- - - [ ] Describe the GitHub flow
+- - - [x] Describe the GitHub flow
```
Comment (Flaigor) [inline README.md:16]: Test 3

File: README.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 We are [Marcos](https://github.com/coppolaop) and [Igor](https://github.com/flaigor)
```
Comment (Flaigor) [inline README.md:2]: Look this

**Review by Flaigor — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (coppolaop) [reply README.md:2]: Nice

File: README.md (hunk 3 above)
Comment (coppolaop) [reply README.md:16]: Test approved

**Review by Flaigor — APPROVED** [review]
thats so close!

**Pull request merged**